### DISCUSSION TITLE
Keyboard Shortcut to Show and Hide Panel

### DISCUSSION THREAD

**Discussion by protean-onion** [TriliumNext/Trilium#3740] [Ideas]
Hello,

While the available keyboard shortcuts are great and encompass Trilium features adequately, it seems it is missing something. Namely, the shortcut to toggle the side panel. It's very tedious to have to raise your fingers from the keyboard, move it over the mouse or the trackpad, and then open it. It would be great if, instead, we could just toggle the side panel with the press of a button combination.

If this is easy to implement, could you please guide me as to how to do it? If not, could it be implemented in a future version of Trilium?

Thanks for your time.

**Comment by zadam** [comment]
Hi, there is a possibility to toggle the panel, just doesn't have a shortcut assigned by default:

![image](https://user-images.githubusercontent.com/617641/226212231-18fd03d0-678e-47f4-827f-2cfc8143ed94.png)

**Reply by protean-onion** [reply]
Thanks!
